custom/echis_reports/ucr/data_sources/kebele_population_profile_data_source.json

```diff
@@ -0,0 +1,1237 @@
+         },
+         {
+            "type":"or",
+            "filters":[
+               {
+                  "operator":"eq",
+                  "expression":{
+                     "datatype":null,
+                     "type":"property_name",
+                     "property_name":"type"
+                  },
+                  "type":"boolean_expression",
+                  "comment":null,
+                  "property_value":"family"
+               },
+               {
+                  "type":"and",
+                  "filters":[
+                     {
+                        "operator":"eq",
+                        "expression":{
+                           "datatype":null,
+                           "type":"property_name",
+                           "property_name":"type"
+                        },
+                        "type":"boolean_expression",
+                        "comment":null,
+                        "property_value":"client"
+                     },
+                     {
+                        "operator":"not_eq",
+                        "type":"boolean_expression",
+                        "expression":{
+                           "datatype":null,
+                           "type":"property_name",
+                           "property_name":"is_member_moving_out_of_the_kebele"
+                        },
+                        "comment":null,
+                        "property_value":"yes"
+                     }
+                  ]
+               },
+               {
+                  "operator":"eq",
+                  "type":"boolean_expression",
+                  "expression":{
+                     "datatype":null,
+                     "type":"property_name",
+                     "property_name":"type"
+                  },
+                  "comment":null,
+                  "property_value":"pregnancy_pp"
+               },
+               {
+                  "operator":"eq",
+                  "type":"boolean_expression",
+                  "expression":{
+                     "datatype":null,
+                     "type":"property_name",
+                     "property_name":"type"
+                  },
+                  "comment":null,
+                  "property_value":"one_to_five"
+               },
+               {
+                  "operator":"eq",
+                  "type":"boolean_expression",
+                  "expression":{
+                     "datatype":null,
+                     "type":"property_name",
+                     "property_name":"type"
+                  },
+                  "comment":null,
+                  "property_value":"one_to_thirty"
+               }
+            ]
+         }
+      ]
+   },
+ 
+   "description":"",
+   "referenced_doc_type":"CommCareCase",
+   "table_id":"static-kebele_population_profile_data_source",
+   "display_name":"Kebele Population Profile Data Source",
+   "named_expressions":{
+   },
+ 
+   "sql_column_indexes":[
```

Comment:
I did notice this but I saw that this empty json elements are added on the static reports that you guys did it previously. fearing that it might crush the app I left it there. I will remove that and push now.